docker/redis.persistent.conf

```diff
@@ -0,0 +1,17 @@
+# Redis configuration for Lisk Service.
+#
+# This config enables persistence
+bind 0.0.0.0
+port 6379
+tcp-backlog 127
+timeout 0
+tcp-keepalive 300
+
+# Snapshotting
+appendonly yes
```

Comment:
```suggestion
append-only yes
```

Reply:
This setting comes from a 3rd party software, it is not possible to change it.